src/PowerShellEditorServices/Services/TextDocument/Handlers/CompletionHandler.cs

```diff
@@ -202,62 +202,65 @@ public void SetCapability(CompletionCapability capability)
             string completionText = completionDetails.CompletionText;
             InsertTextFormat insertTextFormat = InsertTextFormat.PlainText;
 
-            if ((completionDetails.CompletionType == CompletionType.Variable) ||
-                (completionDetails.CompletionType == CompletionType.ParameterName))
+            switch (completionDetails.CompletionType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4801872)